Adyen.EcommLibrary/Model/AdditionalData/InvoiceLine.cs

```diff
@@ -1,17 +1,17 @@
-﻿using Adyen.EcommLibrary.Model.Enum;
-
-namespace Adyen.EcommLibrary.Model.AdditionalData
-{
-    public class InvoiceLine
-    {
-        public long VatAmount { get; set; }
-        public string CurrencyCode { get; set; }
-        public string Description { get; set; }
-        public long ItemAmount { get; set; }
-        public long ItemVatAmount { get; set; }
-        public long ItemVatPercentage { get; set; }
-        public int NumberOfItems { get; set; }
-        public VatCategory VatCategory { get; set; }
-        public string ItemId { get; set; }
-    }
-}
+﻿using Adyen.EcommLibrary.Model.Enum;
+
+namespace Adyen.EcommLibrary.Model.AdditionalData
+{
+    public class InvoiceLine
+    {
+        public long VatAmount { get; set; }
+        public string CurrencyCode { get; set; }
+        public string Description { get; set; }
+        public long ItemAmount { get; set; }
+        public long ItemVatAmount { get; set; }
+        public long ItemVatPercentage { get; set; }
+        public int NumberOfItems { get; set; }
+        public VatCategory VatCategory { get; set; }
+        public string ItemId { get; set; }
+    }
+}
```

Comment:
what is changed here ?

Reply:
Not sure why github decided that we change something